python/dgl/propagate.py

```diff
@@ -57,14 +58,14 @@ def prop_edges(graph,
 def prop_nodes_bfs(graph,
                    source,
                    reverse=False,
-                   message_func='default',
-                   reduce_func='default',
-                   apply_node_func='default'):
+                   message_func=None,
+                   reduce_func=None,
+                   apply_node_func=None):
```

Comment:
What does `None` mean?

Reply:
None is do nothing.
It seems message_func and reduce_func should not have default value of None

Reply:
agree